contracts/TokenSoftTokenV2.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity 0.5.16;
+
+import "./TokenSoftToken.sol";
+import "./capabilities/Blacklistable.sol";
+
+contract TokenSoftTokenV2 is TokenSoftToken, Blacklistable {
+
+  // ERC1404 Error codes and messages
+  uint8 public constant FAILURE_BLACKLIST = 3;
+  string public constant FAILURE_BLACKLIST_MESSAGE = "The transfer was restricted due to blacklist configuration.";
```

Comment:
i wonder if moving failure codes and messages into a map would make upgrades easier because less unstructured state?

Reply:
That could have been a good move in the beginning.  Leaving as is now I think.